packages/@webex/plugin-meetings/test/unit/spec/meeting/index.js

```diff
@@ -8006,8 +8050,10 @@ describe('plugin-meetings', () => {
             sipUri: test1,
             meetingNumber: '12345',
             meetingJoinUrl: url2,
+            permissionToken:
+              'eyJhbGciOiJIUzI1NiJ9.eyJleHAiOiIxMjM0NTYiLCJwZXJtaXNzaW9uIjp7InVzZXJQb2xpY2llcyI6eyJhIjp0cnVlfX19.wkTk0Hp8sUlq2wi2nP4-Ym4Xb7aEUHzyXA1kzk6f0V0',
             owner: test2,
-            permissionToken: 'abc',
+            selfUserPolicies: {a: true},
```

Comment:
What does this property do?

Reply:
We are receiving this in the response. Since the stubbing is happening differently, some method that should have been stubbed is not stubbed and we are also getting this property. This is not breaking anything.